mendix/widgets-resources
# [WC-155] Add style exclusion variables to Atlas web (2nd attempt)

## General
Previous PR: #507 

## Why?
MX devs might want to reuse just parts of Atlas. With style exclusion variables they can choose which styles to exclude.

## What?
- Add exclusion variables for widget default styles and their helper classes.
- Add default values for exclusion variables in `atlas_core` to prevent compilation failures and keep if statements simple.
- Wrap all styles in mixins to conditionally import them in the `atlas_core/main.scss` file.
- Add conditional imports for all styles by evaluating the corresponding exclusion variables.
- When default widgets styles are excluded, their helpers will always be excluded as well. No matter the value of the helper exclusion variable.
- Rename files to singular form and make use of hyphens to make it more readable.

## How to test?
- Test whether exclusion variables disable their corresponding widget and helper styles.
- Try to remove variables from `theme/web/exclusion-variables.scss` and verify that `atlas_core` still compiles. Also check that the corresponding styles are still enabled.
